spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/observation/ObservationAutoConfigurationTests.java

```diff
@@ -108,7 +128,7 @@ void autoConfiguresObservationHandlers() {
 		this.contextRunner.withUserConfiguration(ObservationHandlers.class).run((context) -> {
 			ObservationRegistry observationRegistry = context.getBean(ObservationRegistry.class);
 			List<ObservationHandler<?>> handlers = context.getBean(CalledHandlers.class).getCalledHandlers();
-			Observation.start("test-observation", observationRegistry);
+			Observation.start("test-observation", observationRegistry).stop();
```

Comment:
As far as I can tell, this change is unrelated to wrapping the `DefaultMeterObservationHandler` in a `TracingAwareMeterObservationHandler`. If I've got that right, it should be handled separately please.